Misc/NEWS.d/next/Build/2019-03-29-14-29-06.bpo-36478.hzyneF.rst

```diff
@@ -0,0 +1 @@
+Use C89 for loops in backported pickle patch.  Patch by Anthony Sottile.
```

Comment:
Would you mind to elaborate the rationale to include "gnu89" C standard and GCC 4.8 version? For example:

"Fix compatibility with ISO C89 needed by "gnu89" standard of GCC 4.8: use C89 for loops in backported pickle patch. Patch by Anthony Sottile."

Modify also the commit message, just reuse the same text (without your name, which is already provided by Git).

Reply:
done! thanks for the suggestion :)